lib/utils.js

```diff
@@ -0,0 +1,18 @@
+export function escapeRegexString(regexString) {
+  // See: http://stackoverflow.com/a/6969486/1463681
+  return regexString.replace(/[-[\]/{}()*+?.\\^$|]/g, '\\$&');
```

Comment:
I know you fixed the escape chars issue due to eslint here and thank you for that. But can you please revert the regex here. I want to push this regex change with good amount of testing and testcases so you can give another pr with only the regex change.

Reply:
I found a better solution: https://lodash.com/docs#escapeRegExp
no need for testing this way